src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
```

Comment:
Container in this task does nothing. Let's have a rule for the future, that we never add anything to the code, that does nothing. Remove this tag and styles for the class container